dockers/docker-database/http-server

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python3
+#
+###########################################################################
+# http-server
+# 
+# Python HTTP Server service.
+# This service runs based on the configuration provided in
+# device/<platform>/chassisdb.conf
+#
+# Configuration parameters provided by chassisdb.conf are as follows:
+#
+# start_http_server: if "yes" or "1", http-server will be started on this
+#     node
+# chassis_db_address   : IP address for http-server. This is the same IP used
+#  for the chassis db server
+# http_server_port : Port to bind to, default: 8000
+# http_server_dir  : HTTP server home directory path, default: /var/www/
+#
+###########################################################################
+# Copyright (c) 2021-2022 Cisco Systems, Inc. and its affiliates.
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+###########################################################################
+
+# Example of config:
+'''
+start_http_server=yes
+chassis_db_address=127.0.0.10
+http_server_port=8000
+http_server_dir=/var/www/tftp/
+'''
+
+import argparse
+import http.server
+import os
+import socketserver
+import subprocess
+import syslog
+
+from pathlib import Path
+
+HTTP_DEFAULT_BIND_PORT = "8000"
```

Comment:
>8000 [](http://example.com/codeflow?start=26&length=4)

Do we have rules to protect this service?

Reply:
I will investigate more on this and respond/implement. Thanks

Reply:
@qiluo-msft your point is to have Control plane acl (ip table rules)   to explicitly permit this ?

Reply:
yes, I believe you also want to limit the access, for example by some rules on 5-tuple

Reply:
@abdosi , @qiluo-msft , this port is on the internal network. Do we have any rules on other internal ip/ports which I can refer to?

Reply:
Hi @qiluo-msft , Abhishek and I had a discussion on this. The current Sonic implementation does not have any rule defined for the midplane subnet yet and we need to have a rule for all kinds of access in the midplane subnet. We will have to define platform subnet parameters that caclmgrd could use to define the rules for midplane access for all kinds of midplane traffic in the chassis. I will open a case and we can use a different PR to define the general subnet access rule. Let me know if that is fine.

Reply:
@qiluo-msft on chassis setup we have additional interface `eth1-midplane` both on supervisor and LC for internal traffic (eg: redis and http)  . We will add iptable rule to allow all traffic on that interface. That way http traffic will be protected from outside network (`eth0`) and will get permit only if coming from internal network. I will create separate for that

Reply:
Created PR : https://github.com/sonic-net/sonic-host-services/pull/42 . With this PR any port 8000 traffic that is not coming on eth1-midplane will get drop and thus protected from external network access.